app/assets/javascripts/application.js

```diff
@@ -433,7 +433,6 @@ function reloadOnAjaxComplete(element) {
   tfm.tools.hideSpinner();
   tfm.tools.activateTooltips();
   activate_select2(':root');
-  tfm.numFields.initAll();
```

Comment:
This is probably still needed in some plugins that use the counter field component